tests/extensions/tabular/test_panda_tools.py

```diff
@@ -9,24 +9,27 @@ class TestTabularPandaTools:
 
     def test_data_from_dateframe(self):
         with open(os.path.join(BASE, 'files', 'test.csv')) as fp:
-            headers, data = panda_tools.csv_pandas(fp)
+            sheets = panda_tools.csv_pandas(fp)
 
-        assert type(data) == list
-        assert type(data[0]) == dict
+        sheet = sheets.popitem()[1]
+        assert type(sheet[0]) == list
+        assert type(sheet[0][1]) == dict
 
     def test_csv_pandas(self):
         with open(os.path.join(BASE, 'files', 'test.csv')) as fp:
-            headers, data = panda_tools.csv_pandas(fp)
+            sheets = panda_tools.csv_pandas(fp)
 
-        assert headers[0] == {'field': 'one', 'id': 'one', 'name': 'one', 'sortable': True}
-        assert data[0] == {'one': 'à', 'two': 'b', 'three': 'c'}
+        sheet = sheets.popitem()[1]
+        assert sheet[0][0] == {'field': 'one', 'id': 'one', 'name': 'one', 'sortable': True}
+        assert sheet[1][0] == {'one': 'à', 'two': 'b', 'three': 'c'}
 
     def test_tsv_pandas(self):
         with open(os.path.join(BASE, 'files', 'test.tsv')) as fp:
-            headers, data = panda_tools.csv_pandas(fp)
+            sheets = panda_tools.csv_pandas(fp)
 
-        assert headers[0] == {'field': 'one\ttwo\tthree', 'id': 'one\ttwo\tthree', 'name': 'one\ttwo\tthree', 'sortable': True}
-        assert data[0] == {'one\ttwo\tthree': 'a\tb\tc'}
+        sheet = sheets.popitem()[1]
+        assert sheet[0][0] == {'field': 'one\ttwo\tthree', 'id': 'one\ttwo\tthree', 'name': 'one\ttwo\tthree', 'sortable': True}
+        assert sheet[1][0] == {'one\ttwo\tthree': 'a\tb\tc'}
 
     # def test_dta_pandas():
```

Comment:
Uncertain. `.dta` isn't currently a supported filetype, judging from https://github.com/CenterForOpenScience/modular-file-renderer/blob/develop/setup.py#L647 being commented out. If support for it is re-enabled, this test should be used and updated. This wasn't directly associated with this PR, though, so I left it alone.